Weekly Changelog 2023-05-01 to 2023-05-08 (#65538)
CleverRaven/Cataclysm-DDA: data/changelog.txt

Add option to hide bionics
Allow JSON monster special attacks to use the dialog condition system
Confirmation before attacking neutral mobs

> Subtle spell and silent spell tweaks

Reviewer: ```suggestion
```

> Subtle spell and silent spell tweaks
> Unhardcode and rework grabs, ranged pulls

Reviewer 2: ```suggestion
Unhardcode and rework grabs, ranged pulls
EOCs can have context vars that can be passed into nested EOCs
```

Add new weather types: mist and fog
Allow players to keep bees
Apartment complex: parking garage, lobby variants, roof additions
Improved nether monster corpse

> Added a new larger shelter kit with its recipes

Reviewer: ```suggestion
```

Crustacean Mutation tree
Dead trees can be harvested for wood(once per tree)

> map update: bike shop

Reviewer: ```suggestion
```

> Improved nether monster corpse
> Added a new larger shelter kit with its recipes
> Crustacean Mutation tree
> Dead trees can be harvested for wood(once per tree)
> map update: bike shop
> Integrated AR system CBM
> map update: dive shop

Reviewer 2: ```suggestion
Crustacean mutation tree
Dead trees can be harvested for wood(once per tree)
```

> map update: dive shop

Reviewer: ```suggestion
```

[Magiclysm] Adds several new professions to the Magiclysm mod
[Bombastic Perks] Added resurrecting meat monstrosities to bombastic perks
[Bombastic Perks] Forcefield and Evasion enchants

> Aftershock: Elemental bionic weapons.
> Adds the recycler perk
> [Xedra Evolved] Revamped the inventor class

Reviewer 2: ```suggestion
[Aftershock] Elemental bionic weapons.
[Bombastic Perks] Adds the recycler perk
[Xedra Evolved] Revamped the inventor class
```

partial skill levels contribute to most game tests
Portal Storm Coherency Pass
Stop zomborgs from exploding on death

> Change anvil volume to a reasonable 24L

Reviewer: ```suggestion
```

> Change anvil volume to a reasonable 24L
> Fix bird chicks and playing with corvids
> Plant distribution tweaks

Reviewer 2: ```suggestion
```

> Plant distribution tweaks

Reviewer: ```suggestion
```

Fix energy guns(AFS) on NPC
Faction camps now distribute calories based on actual calories and not default calories
Fix calculation for inserting into nested containers
Cooking raw food once again improves their calories
Fix crash when monster attacks a dead NPC

> [Magiclysm] Fix the diviner's swamp dragon quest

Reviewer: ```suggestion
```

> mutagen chelator doesn't turn you into a crab

Reviewer: ```suggestion
```

Make copy-from copy terrain/furniture examine actions
Fix crash when installing vehicle parts on empty square
Fix a crash related to NPC death
Prevent mission marker from being cutoff in the overmap

> Fix infinite recursion when trying to drain battery

Reviewer: ```suggestion
```

> Cooking raw food once again improves their calories
> Fix crash when monster attacks a dead NPC
> [Magiclysm] Fix the diviner's swamp dragon quest
> mutagen chelator doesn't turn you into a crab
> Make copy-from copy terrain/furniture examine actions
> Fix crash when installing vehicle parts on empty square
> Fix a crash related to NPC death
> Prevent mission marker from being cutoff in the overmap
> Fix infinite recursion when trying to drain battery
> Exodii will now properly be mad at you if you steal all their resources from stone barns

Reviewer 2: ```suggestion
Make copy-from copy terrain/furniture examine actions
Prevent mission marker from being cutoff in the overmap
Exodii will now properly be mad at you if you steal all their resources from stone barns
```

Modernize string_formatter
Migrate some JSON APIs to string_view
Add units::temperature_delta

> EOCs can have Context Vars
> Run EOC with passed in variables
> Upgrade clang-tidy used in CI to LLVM 16

Reviewer 2: ```suggestion
Upgrade clang-tidy used in CI to LLVM 16
```
